db/db_impl_write.cc

```diff
@@ -1234,6 +1268,7 @@ Status DBImpl::ThrottleLowPriWritesIfNeeded(const WriteOptions& write_options,
 }
 
 Status DBImpl::ScheduleFlushes(WriteContext* context) {
+  // TODO (yanqin) add invocation of FlushManager
```

Comment:
Not necessary, because in this case the column families are chosen already. Hence, nothing to pick here.

Reply:
I came across this and realize that this is different from what we discussed earlier this afternoon. Could you confirm whether you need a callback here? If so, what information you need?